src/raftstore/store/util.rs

```diff
@@ -229,6 +229,28 @@ pub fn get_region_approximate_size(db: &DB, region: &metapb::Region) -> Result<u
     Ok(size)
 }
 
+/// Get the approximate number of records in the region.
+pub fn get_region_approximate_count(db: &DB, region: &metapb::Region) -> Result<u64> {
+    let cf = rocksdb_util::get_cf_handle(db, CF_WRITE)?;
+    let start = keys::enc_start_key(region);
+    let end = keys::enc_end_key(region);
+    let range = Range::new(&start, &end);
+    let (mut count, _) = db.get_approximate_memtable_stats_cf(cf, &range);
+    let collection = db.get_properties_of_tables_in_range(cf, &[range])?;
+    for (_, v) in &*collection {
+        let props = RowsProperties::decode(v.user_collected_properties())?;
+        count += props.get_approximate_rows_in_range(&start, &end);
+    }
+    Ok(count)
+}
+
+/// Get the approximate number of records and size in the region.
+pub fn get_region_approximate_stats(db: &DB, region: &metapb::Region) -> Result<(u64, u64)> {
```

Comment:
I prefer defining a Stat structure here instead of the tuple, maybe we may support more stats later.